src/types/types.ts

```diff
@@ -0,0 +1,28 @@
+export interface HttpResponse {
```

Comment:
Something wrong with indentation in this file.

Please add rule to your eslint/prettier to ensure the same indentation for all files

Reply:
Done